clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/controllers/CloudFoundryLoadBalancerController.groovy

```diff
@@ -17,83 +17,78 @@
 package com.netflix.spinnaker.clouddriver.cf.controllers
 
 import com.fasterxml.jackson.annotation.JsonProperty
-import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider
-import com.netflix.spinnaker.clouddriver.cf.model.CloudFoundryLoadBalancer
-import com.netflix.spinnaker.clouddriver.cf.model.CloudFoundryResourceRetriever
+import com.netflix.spinnaker.cats.cache.Cache
+import com.netflix.spinnaker.cats.cache.CacheData
+import com.netflix.spinnaker.cats.cache.RelationshipCacheFilter
+import com.netflix.spinnaker.clouddriver.cf.cache.Keys
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestMethod
 import org.springframework.web.bind.annotation.RestController
 
+import static com.netflix.spinnaker.clouddriver.cf.cache.Keys.Namespace.LOAD_BALANCERS
+
 @RestController
 @RequestMapping("/cf/loadBalancers")
 class CloudFoundryLoadBalancerController {
 
-  @Autowired
-  AccountCredentialsProvider accountCredentialsProvider
+  private final Cache cacheView
 
   @Autowired
-  CloudFoundryResourceRetriever cfResourceRetriever
+  CloudFoundryLoadBalancerController(Cache cacheView) {
+    this.cacheView = cacheView
+  }
 
   @RequestMapping(method = RequestMethod.GET)
   List<CloudFoundryLoadBalancerAccount> list() {
-    getSummaryForLoadBalancers(cfResourceRetriever.loadBalancersByAccount).values() as List
+    Collection<String> loadBalancers = cacheView.getIdentifiers(LOAD_BALANCERS.ns)
```

Comment:
I think this will suffer the same problem that the AWS version does in that in a multi cats enabled cloudprovider world the CacheView is an aggregate of all the cloudprovider data, so getIdentifiers(LOAD_BALANCERS.ns) is going to return results from all providers.

I was going to add a Keys.getCloudProviderKey() that would essentially return provider:* (in this case `cf:*`) so that this could change to `cacheView.filterIdentifiers(LOAD_BALANCERS.ns, Keys.getCloudProviderKey())` 

That said, I don't know that we actually have anything that exercises this method besides tests and I think there is a proposal on the table to cull all the unused bits out of here from @ttomsu which might also resolve this specific issue